### PR TITLE
magnus changes for FCI

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Comment by aboba** [comment]
Magnus's suggestion is more comprehensive than the original text, so I'm ok with taking this change.

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -5871,12 +5871,34 @@ a=msid:-
...
-          defined in <xref target="I-D.ietf-avtext-lrr" />, for each
-          FCI entry where the SSRC is found in the outgoing SSRC
-          table, deliver a copy of the RTCP packet to the "m=" line
-          associated with that SSRC.</t>
+          <t> Other feedback messages include the target SSRC in the
+          Feedback Control Information (FCI). Here, each FCI needs to
+          be processed and the SSRC field identified. Also, the
+          individual FCI, combined with the RTCP packet header context,
+          needs to be forwarded to the "m=" line associated with
+          that SSRC. Examples of this pattern are:
```
Comment (fluffy) [inline draft-ietf-rtcweb-jsep.xml]: I would change "Examples of this pattern are:" to "This includes the following:"

Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: Again this needs to be imperative to match the rest of the text.

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -5871,12 +5871,34 @@ a=msid:-
...
+		    target="RFC5104"/> (PT=PSFB, FMT=4).</t>
+
+			<t hangText="Temporal-Spatial Trade-off Request (TSTR):"><xref
+			target="RFC5104"/> (PT=PSFB, FMT=5).</t>
+
+			<t hangText="Temporal-Spatial Trade-off Notification
+			(TSTN):">This <xref target="RFC5104"/> defined message
+			(PT=PSFB, FMT=5) is actually a notification in response to
+			a TSTR this endpoint sent using the SSRC the FCI
+			identifies as source. </t>
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: as the source?

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -5871,12 +5871,34 @@ a=msid:-
...
+
+			<t hangText="H.271 Video Back Channel Message
+			(VBCM):"><xref target="RFC5104"/> (PT=PSFB, FMT=7).</t>
+
+			<t hangText="Layer Refresh Request (LRR):"><xref
+			target="I-D.ietf-avtext-lrr"/> (PT=PSFB, FMT=TBD). </t>
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: Nit, remove space between . and <

**Comment by juberti** [comment]
This looks good but needs to be massaged to match the surrounding text.

**Comment by fluffy** [comment]
+1

**Pull request merged**